docs/source/what-is-mlflow.rst

```diff
@@ -0,0 +1,51 @@
+
+MLflow is a versatile, expandable, open-source platform for managing machine learning workflows and artifacts across the machine learning lifecycle. It has predefined flavors for many popular ML libraries, but can be used with any library, algorithm, or deployment tool. It is designed to be extensible, so you can write plugins to support new workflows, libraries, and tools.
+
+.. figure:: _static/images/what-is-mlflow/mlflow-overview.png
+    :width: 100%
+
+MLflow is composed of five components:
+
+- `MLflow Tracking <https://www.mlflow.org/docs/latest/tracking.html>`_: An API and UI for logging parameters, code versions, metrics, and output files when running your machine learning code and for later visualizing the results.
+
+- `MLflow Projects <projects>`_: A standard format for packaging reusable data science code that can be run with different parameters to train models, visualize data, or perform any other data science task.
+
+- `MLflow Models <models>`_: A model packaging format and tools that let you easily deploy the same model (from any ML library) to batch or real-time scoring on platforms such as Docker, Apache Spark, Azure ML and AWS SageMaker.
+
+- `MLflow Registry <registry>`_: A centralized model store, set of APIs, and UI, to collaboratively manage the full lifecycle of MLflow Models.
+
+- `MLflow Recipes <recipes>`_: Templates and an API for combining MLflow Tracking, Projects, and Models to create reusable workflows.
+
+What is MLflow used for?
+------------------------
+
+MLflow is used to manage the machine learning lifecycle from initial model development through deployment. It combines:
+
+- Tracking experiments to record and compare parameters and results (MLflow Tracking)
+- Packaging ML code in a reusable, reproducible form in order to share with other data scientists or transfer to production (MLflow Projects)
+- Managing and deploying models from a variety of ML libraries to a variety of model serving and inference platforms (MLflow Models)
+- Collaboratively managing a central model store, including model versioning, stage transitions, and annotations (MLflow Registry)
+- Creating and speeding iterative development with templates and reusable scripts (MLflow Recipes)
+
+Who uses MLflow?
+----------------
+
+MLflow is used by Data Scientists and Machine Learning Engineers as well as by MLOps and IT professionals. Data Scientists and Machine Learning Engineers use MLflow to track experiments, structure code, package models, and to review and select models for deployment. MLOps and IT professionals use MLflow to securely deploy models to production and to monitor deployments for performance and accuracy. 
+
+MLflow use-cases
+----------------
+
+MLflow is used for a variety of machine learning use-cases. Here are some examples:
+
+- A Data Scientist beginning work with a new project structures their code as an MLflow Project so that they can easily share it with others and run it with different parameters.
```

Comment:
```suggestion
- A Data Scientist who is beginning work to on a new project structures their code as an MLflow Project so that they can easily share it with others and run it with different parameters.
```